Extension/src/LanguageServer/client.ts

```diff
@@ -3506,12 +3510,39 @@ export class DefaultClient implements Client {
                     character: range.end.character,
                     line: range.end.line
                 }
-            }
+            },
+            copyToClipboard: copy ?? false
         };
 
         const result: CreateDeclarationOrDefinitionResult = await this.languageClient.sendRequest(CreateDeclarationOrDefinitionRequest, params);
-        // TODO: return specific errors info in result.
-        if (result.changes === undefined) {
+        // Create/Copy returned no result.
+        if (result.edit === undefined) {
+            await vscode.window.showInformationMessage(result.errorText); // Copy/Create Declaration/Definition was completely unsuccessful due to api failure.
```

Comment:
Another `await` of `showInformationMessage`, which probably isn't desirable.

Reply:
So, we should only use await when the code that follows needs the result, right? So the other awaits below should be removed too?

Reply:
Or await may be needed to ensure the ordering of calls (A completes before B is called).  I'm not familiar with the other calls here being awaited.  I just noticed these UI calls specifically as they won't complete until/unless the user actually actions on them, so can cause indefinite stalls.